erpnext_shopify/shopify_requests.py

```diff
@@ -0,0 +1,87 @@
+import frappe
+import json, math
+from frappe import _
+from .exceptions import ShopifyError
+from frappe.utils import get_request_session
+
+def get_shopify_settings():
+	d = frappe.get_doc("Shopify Settings")
+	if d.shopify_url:
+		return d.as_dict()
+	else:
+		frappe.throw(_("Shopify store URL is not configured on Shopify Settings"), ShopifyError)
+
```

Comment:
I would prefer naming such that I can use `shopify_requests.get`, `shopify_requests.post`, etc.